fawkes/fetch/fetch.py

```diff
@@ -62,6 +63,10 @@ def fetch_reviews(fawkes_config_file = constants.FAWKES_CONFIG_FILE):
                     reviews = appstore.fetch(
                         review_channel
                     )
+                elif review_channel.channel_type == ReviewChannelTypes.SPLUNK:
```

Comment:
Can you add an example of splunk config in the https://github.com/intuit/fawkes/blob/master/app/sample-mint-config.json

You can keep the `is_channel_enabled` as false, for it to not break the unit tests.

Reply:
Pls feel free to say no, if it doesn't make sense.

Reply:
Added the splunk channel in sample config, with channel_enabled=false